js/autosleep-service.js

```diff
@@ -0,0 +1,46 @@
+(function() {
+    'use strict';
+
+    function AutoSleepService($interval, $filter) {
+        var service = {};
+        var autoSleepTimer;
+
+        service.startAutoSleepTimer = function($scope) {
+            if (typeof config.autoTimer !== 'undefined' && typeof config.autoTimer.autosleep !== 'undefined' && typeof config.autoTimer.autowake !== 'undefined') {
+                service.scope = $scope;
```

Comment:
Services should not manipulate the scope in any way, that's the job of the controller. The controller should be asking the Service to put the monitor to sleep/wake it up and the service should be executing the actual command.